packages/documentation/docs/controls/_pill_styleguide.md

```diff
@@ -0,0 +1,52 @@
+Pills are components used to display a small pieces of information in a compact and visually appealing way. Typically pills contain a concise label and sometimes an icon, and they aren't clickable or closable.
+
+
+![Pill overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=1377%3A3110&mode=design&t=ZmcRP4ggXtr8b7vZ-1)
+
+1. Container
+2. Icon
+3. Label text
+
+## Variants
+With our pill variants, you can apply different colors based on their purpose, importance or context. We use chip variants to show class, status and levels of importance. The custom variant is often used for pills that visualize a high number of different categories.
+
+Pill variants:
+
+- **Primary**: For high visual emphasis.
+- **State-related variants**: Alarm, critical, warning, success, info and neutral.
+- **Custom**: For a customized background and label color.
+
+![Pill variants](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=1375%3A1985&mode=design&t=ZmcRP4ggXtr8b7vZ-1)
+
+## Options
+
+- **Align left**: Position the pill content to the left side.
+- **Background**: Use to set a custom background color when you require more flexibility in styling the pill. Only available for the custom pill variant.
+- **Color**: Customize font and icon color for pill. This allows users to specify a unique font color in combination with a custom background color (only applicable when the variant is set to 'custom').
+- **Icon**: Pills can include a close icon within the element which is positioned before the pills label.
+- **Outline**: Use for lower visual emphasis.
+- **Width**: Typically content length determines pill width with a minimum width of '2rem'. Pill width can be set to a specific value.
+
+## Behavior
+
+- **Placement**: We usually position pills inline with other elements to convey their status or category. We do not place pills within input and filter components, as these have their own version known as the input chip. It is possible to add components similar to pills to tabs and navigation menu items. These counter or notification components are provided as component options. 
```

Comment:
Not sure if we should provide a name here (input chip) since people will start looking for it and the actual component is called filter-chip and not public as of now.

Reply:
fixed in bfd8eb5a1587b42e87f37daa83677df1d92eba07